src/aind_data_transfer/transformations/ome_zarr.py

```diff
@@ -788,4 +781,203 @@ def _get_axes_5d(time_unit: str = "millisecond", space_unit: str = "micrometer")
         {"name": "y", "type": "space", "unit": f"{space_unit}"},
         {"name": "x", "type": "space", "unit": f"{space_unit}"},
     ]
-    return axes_5d
\ No newline at end of file
+    return axes_5d
+
+
+def create_pyramid(
+    arr: ArrayLike,
+    n_lvls: int,
+    scale_factors: tuple,
+    chunks: Union[str, tuple] = "preserve",
+) -> list:
+    """
+    Create a multiscale pyramid of the input data.
+
+    Parameters
+    ----------
+    arr : array-like
+        The input data to be downsampled.
+    n_lvls : int
+        The number of pyramid levels to generate.
+    scale_factors : tuple
+        The scale factors for downsampling along each dimension.
+    chunks : tuple
+        The chunk size to use for the output arrays.
+
+    Returns
+    -------
+    list
+        A list of Dask arrays representing the pyramid levels.
+    """
+    pyramid = multiscale(
```

Comment:
@carshadi, have you noticed aliasing in the downsampled resolutions? As far as I am concerned, we haven't checked this.

Reply:
I haven't noticed aliasing, but we could open a separate issue for that, unless it's related to these changes?